ql/cashflows/cpicoupon.hpp

```diff
@@ -191,6 +191,8 @@ namespace QuantLib {
 
         ext::shared_ptr<ZeroInflationIndex> cpiIndex() const;
 
+        virtual Real indexFixing() const override;
```

Comment:
This can be just
```suggestion
        Real indexFixing() const override;
```
for consistency with the other methods.

Reply:
ok